gradle.properties

```diff
@@ -4,7 +4,7 @@ org.gradle.jvmargs=-Xmx2G -XstartOnFirstThread
 # Fabric Properties
 	# check these on https://fabricmc.net/use
 	minecraft_version = 1.8.9
-	yarn_mappings = 1.8.9+build.202009070256
+	yarn_mappings = 1.8.9+build.202009230433
```

Comment:
![image](https://user-images.githubusercontent.com/25501603/93980959-a9a70700-fdc2-11ea-95f1-fbff47157767.png)

Reply:
that will require replacing every single entity method in the mod with `method_xxxx`

Reply:
lmao